plugin-maven/README.md

```diff
@@ -341,6 +342,53 @@ Spotless requires Maven to be running on JRE 8+.
 </antlr4formatter>
 ```
 
+<a name="sql-dbeaver"></a>
+<a name="applying-dbeaver-to-sql-scripts"></a>
+
+## SQL
+
+[code](https://github.com/diffplug/spotless/blob/main/plugin-maven/src/main/java/com/diffplug/spotless/maven/sql/Sql.java). [available steps](https://github.com/diffplug/spotless/tree/main/plugin-maven/src/main/java/com/diffplug/spotless/maven/sql).
+
+```xml
+<configuration>
+  <sql>
+    <!-- You have to set the target manually -->
+    <includes>
+      <include>src/main/resources/**/*.sql</include>
+    </includes>
+
+    <dbeaver /> <!-- has its own section below -->
+
+    <licenseHeader>
+      <content>-- (C)$YEAR</content>  <!-- or <file>${basedir}/license-header</file> -->
+    </licenseHeader>
```

Comment:
In the [gradle docs for sql](https://github.com/diffplug/spotless/tree/main/plugin-gradle#sql), we left the license step out completely, based on the assumption that people don't put license headers on their sql.  If you are using the license step, feel free to leave this section in.  If you don't plan to use the license step on your sql, then I would leave it out.

If you choose the leave the license docs in, you'll need to copy the correct docs from the [typescript section](https://github.com/diffplug/spotless/tree/main/plugin-maven#typescript).  The sql docs are currently incorrect, because the default delimiter is null.

With or without a license step, be sure to add a `<prettier /> <!-- has its own section below -->` line, because prettier has some sql-formatting plugins.